qutebrowser/mainwindow/mainwindow.py

```diff
@@ -54,44 +54,59 @@ def get_window(via_ipc, force_window=False, force_tab=False,
     """
     if force_window and force_tab:
         raise ValueError("force_window and force_tab are mutually exclusive!")
+
     if not via_ipc:
         # Initial main window
         return 0
-    window_to_raise = None
+
+    open_target = config.get('general', 'new-instance-open-target')
+
+    # Apply any target overrides, ordered by precedence
     if force_target is not None:
         open_target = force_target
-    else:
-        open_target = config.get('general', 'new-instance-open-target')
-    if (open_target == 'window' or force_window) and not force_tab:
+    if force_window:
+        open_target = 'window'
+    if force_tab and open_target == 'window':
+        open_target = 'tab'  # for lack of a better default
```

Comment:
Well, picking "tab" as opposed to "tab-bg" or "tab-silent" is essentially arbitrary in this case, but "tab" is picked since it makes the least assumptions about the scenario in which force_tab was used. (When does that ever actually happen?)